examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongCommand.java

```diff
@@ -4,6 +4,8 @@
 import com.matter.controller.commands.common.CredentialsIssuer;
 
 public final class PairOnNetworkLongCommand extends PairingCommand {
+  private final int mPort = 5540;
```

Comment:
const should be static, and name should be capital,  please update to 
 private final static int MATTER_PORT = 5540;